src/utils.h

```diff
@@ -1,19 +1,18 @@
-#ifndef _UTILS_H_
-#define _UTILS_H_
+#pragma once
 
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
 
+#define ARRAY_COUNT(array) (sizeof(array) / sizeof(array[0]))
```

Comment:
Is this safe to use for any array?
What happens for an array that is declared, but not initialized? Wouldn't it throw exception when trying to access `array[0]` ?

Reply:
Yes it's a safe and standard C idiom, it will work as long as the size is know at compilation time so no risk for our applications as we don't do dynamic allocation